addons/api/addon/generated/models/channel-recording.js

```diff
@@ -31,7 +31,7 @@ export default class GeneratedChannelRecordingModel extends BaseModel {
       'The total number of errors that occurred during the use of this Channel.',
     readOnly: true,
   })
-  errors;
+  errors_session_recording;
```

Comment:
Can we call this something more generic since this is also for channel and connection recordings?